src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionService.cs

```diff
@@ -42,40 +43,47 @@ internal abstract class AbstractAsyncGoToDefinitionService : AbstractFindDefinit
                 workspace, document.Id, position, virtualSpace: 0, cancellationToken);
         }
 
-        public async Task<INavigableLocation?> FindDefinitionLocationAsync(Document document, int position, CancellationToken cancellationToken)
+        public async Task<(INavigableLocation? location, TextSpan symbolSpan)> FindDefinitionLocationAsync(
+            Document document,
+            int position,
+            CancellationToken cancellationToken)
```

Comment:
main change here is to allow our internal API to return not just hte location, but also the span of hte reference.  NOrmal go-to doesn't need this, but go-to-on-hover uses this to determine waht span to underline.

Reply:
if we don't have an `INavigableLocation` can we do anything with the text span?  Or should this be `(INavigableLocation location, TextSpan symbolSpan)?` where the whole tuple is nullable?